test/testrunner.py

```diff
@@ -123,18 +124,46 @@ def main():
                     f"Instead seeing {frameworks_in_images} frameworks."
                 )
             framework = frameworks_in_images[0]
+
+            # Separate multi-node EKS tests from single-node tests in execution to prevent resource contention
+            multi_node_tests_per_framework = {
+                "mxnet": {
+                    "test_eks_mxnet_multi_node_training_horovod_mnist":
+                        os.path.join(test_path, "mxnet", "training", "test_eks_mxnet_multinode_training.py"),
+                    "test_eks_mxnet_multinode_training":
+                        os.path.join(test_path, "mxnet", "training", "test_eks_mxnet_multinode_training.py"),
+                },
+                "pytorch": {
+                    "test_eks_pytorch_multinode_node_training":
+                        os.path.join(test_path, "pytorch", "training", "test_eks_pytorch_training.py"),
+                },
+                "tensorflow": {
+                    "test_eks_tensorflow_multi_node_training_gpu":
+                        os.path.join(test_path, "tensorflow", "training", "test_eks_tensorflow_multi_node_training.py"),
+                }
+            }
+            test_skip_arg = " and ".join([
+                f"not {test_name}"
+                for _, tests in multi_node_tests_per_framework.items()
+                for test_name, _ in tests.items()
+            ])
+            multi_node_test_location = list({f"{location}::{test_name}" for test_name, location in
+                                             multi_node_tests_per_framework[framework].items()})
+
             eks_cluster_name = setup_eks_cluster(framework)
 
-            #setup kubeflow
+            # setup kubeflow
             eks_utils.setup_kubeflow(eks_cluster_name)
             
-            # Split training and inference, and run one after the other, to prevent scheduling issues
+            # Change 1: Split training and inference, and run one after the other, to prevent scheduling issues
```

Comment:
nit: is there a change 2?

Reply:
Right. Meant to add "Change 2: Separate multi-node EKS tests from single-node tests in execution to prevent resource contention". I moved the comment above this block, but didn't revert the change I made to make it "Change 1". Thanks for pointing it out.